doc_src/cmds/bind.rst

```diff
@@ -7,49 +7,66 @@ Synopsis
 
 .. synopsis::
 
-    bind [(-M | --mode) MODE] [(-m | --sets-mode) NEW_MODE] [--preset | --user] [-s | --silent] [-k | --key] SEQUENCE COMMAND ...
-    bind [(-M | --mode) MODE] [-k | --key] [--preset] [--user] SEQUENCE
-    bind (-K | --key-names) [-a | --all] [--preset] [--user]
+    bind [(-M | --mode) MODE] [(-m | --sets-mode) NEW_MODE] [--preset | --user] [-s | --silent] KEYS COMMAND ...
+    bind [(-M | --mode) MODE] [--preset] [--user] [KEYS]
+    bind [-a | --all] [--preset] [--user]
     bind (-f | --function-names)
     bind (-L | --list-modes)
-    bind (-e | --erase) [(-M | --mode) MODE] [--preset] [--user] [-a | --all] | [-k | --key] SEQUENCE ...
+    bind (-e | --erase) [(-M | --mode) MODE] [--preset] [--user] [-a | --all] | KEYS ...
 
 Description
 -----------
 
-``bind`` manages bindings.
+``bind`` manages key bindings.
 
-It can add bindings if given a SEQUENCE of characters to bind to. These should be written as :ref:`fish escape sequences <escapes>`. The most important of these are ``\c`` for the control key, and ``\e`` for escape, and because of historical reasons also the Alt key (sometimes also called "Meta").
+If both ``KEYS`` and ``COMMAND`` are given, ``bind`` adds (or replaces) a binding in ``MODE``.
+If only ``KEYS`` is given, any existing binding in the given ``MODE`` will be printed.
 
-For example, :kbd:`Alt`\ +\ :kbd:`W` can be written as ``\ew``, and :kbd:`Control`\ +\ :kbd:`X` (^X) can be written as ``\cx``. Note that Alt-based key bindings are case sensitive and Control-based key bindings are not. This is a constraint of text-based terminals, not ``fish``.
+``KEYS`` is a comma-separated list of key names.
+Modifier keys can be specified by prefixing a key name with a combination of ``ctrl-``, ``alt-`` and ``shift-``.
+For example, :kbd:`Alt`\ +\ :kbd:`W` is written as ``alt-w``.
```

Comment:
Are these case-insensitive now or did you end up not doing that?

Reply:
not yet because ctrl doesn't work well in title case, and we'd need to decide what's the canonical version, is it pagedown or PageDown

Reply:
I would add a note here that it's case-sensitive.

And does alt-W then do the same as alt-shift-w?

Reply:
yes, both `alt-W` and `alt-shift-w` are canonicalized to `alt-W`. Actually all keys are eagerly canonicalized; so `bind` will never output duplicates. For example `\t` and `tab` refer to the same binding.
One easy way to find out is to define a binding using way of writing it and the query it with another way.